### ISSUE TITLE
Server Error when captcha empty

### ISSUE THREAD

**Issue by RomainL972** [Meteo-Concept/hcaptcha-bundle#1]
Hello,
I am using this bundle to have a captcha in my contact form, and sometimes I receive a server error : 
```json
{
    "class": "TypeError",
    "message": "Argument 1 passed to MeteoConcept\\HCaptchaBundle\\Form\\HCaptchaResponse::__construct() must be of the type string, null given, called in /home/kreyolgyhm/www/vendor/meteo-concept/hcaptcha-bundle/Form/DataTransformer/HCaptchaValueFetcher.php on line 58",
    "code": 0,
    "file": "/home/kreyolgyhm/www/vendor/meteo-concept/hcaptcha-bundle/Form/HCaptchaResponse.php:33",
    "trace": [
        "/home/kreyolgyhm/www/vendor/meteo-concept/hcaptcha-bundle/Form/DataTransformer/HCaptchaValueFetcher.php:58",
        "/home/kreyolgyhm/www/vendor/symfony/form/Form.php:1085",
        "/home/kreyolgyhm/www/vendor/symfony/form/Form.php:646",
        "/home/kreyolgyhm/www/vendor/symfony/form/Form.php:576",
        "/home/kreyolgyhm/www/vendor/symfony/form/Extension/HttpFoundation/HttpFoundationRequestHandler.php:109",
        "/home/kreyolgyhm/www/vendor/symfony/form/Form.php:490",
        "/home/kreyolgyhm/www/src/Controller/HelpController.php:27",
        "/home/kreyolgyhm/www/vendor/symfony/http-kernel/HttpKernel.php:157",
        "/home/kreyolgyhm/www/vendor/symfony/http-kernel/HttpKernel.php:79",
        "/home/kreyolgyhm/www/vendor/symfony/http-kernel/Kernel.php:196",
        "/home/kreyolgyhm/www/public/index.php:28"
    ]
}
```
It looks like it is caused by the captcha result being absent from the request. I don't know how it happens with my clients but when I delete the captcha widget from the webpage, I receive this error too. Is there anything to do to fix it?

**Comment by lgeorget** [comment]
Hi,

sorry for answering this so late. I don't really know how this particular error can occur, but I guess I should not rely on the Captcha response being present. This DataTransformer is called before any validation takes place so I have to check for the value to be a string.

I guess the error can occur if the HCaptcha's widget doesn't load.

**Comment by RomainL972** [comment]
Hi,
I think it happens when bots try to make my website send emails automatically, so they don't include the captcha in their request.

**Comment by lgeorget** [comment]
Hmm yes, that's one good reason for the captcha not to load.

I've pushed an update to master, would you be willing to try it out?

**Comment by lgeorget** [comment]
Fixed by release v1.1.2 (2ed4eb31641dbaf81acb2a492d549b7526b438c0).

**Comment by RomainL972** [comment]
Thank you very much
